standard/links_stream.lua

```diff
@@ -83,47 +86,135 @@ function StreamLinks.processStreams(forwardedInputArgs)
 		forwardedInputArgs.stream = nil
 	end
 
+	local processedStreams = {}
 	for _, platformName in pairs(StreamLinks.countdownPlatformNames) do
-		local streamValue = Logic.emptyOr(
-			streams[platformName],
-			forwardedInputArgs[platformName],
-			Variables.varDefault(platformName)
+		local streamValues = Table.merge(
+			Table.filterByKey(forwardedInputArgs, StreamLinks.isStream),
+			Table.filterByKey(streams, StreamLinks.isStream)
 		)
```

Comment:
I don't get exactly what the whole function is doing, especially this part since I feel it can be out of the for-loop

Reply:
adjusted it (moved most of it out of the loop
you are right it was quite redundant^^